### ISSUE TITLE
Helm Chart Might Be Missing RBAC Rule

### ISSUE THREAD

**Issue by dpeckett** [seaweedfs/seaweedfs-csi-driver#152]
Installed the `seaweedfs-csi-driver` v1.1.8 Helm chart and attempted to create a Pod and a PVC.

```
Warning  FailedMount             106s (x11 over 7m59s)  kubelet                  MountVolume.SetUp failed for volume "pvc-bf914962-3edc-484e-b01f-d71f968745ed" : rpc error: code = Unknown desc = persistentvolumes "pvc-bf914962-3edc-484e-b01f-d71f968745ed" is forbidden: User "system:serviceaccount:seaweedfs:seaweedfs-csi-driver-node-sa" cannot get resource "persistentvolumes" in API group "" at the cluster scope
```

Was able to fix volume mounting by adding an additional RBAC rule to the `driver-registrar-node-role` ClusterRole:

```yaml
- apiGroups:
  - ""
  resources:
  - persistentvolumes
  verbs:
  - get
  - list
  - watch
  - update
  - patch
 ```

**Comment by spastorclovr** [comment]
I can second that. Same issue solved by adding RBAC rule to driver-registrar-node-role. 
The weirdest thing being that I used to be able to mount PVC without any pb not so long ago without those permissions.  Only when uninstall / install csi driver di I got the pb..

**Comment by chrislusf** [comment]
Someone sends a PR for this?

**Comment by spastorclovr** [comment]
Just did it right here https://github.com/seaweedfs/seaweedfs-csi-driver/pull/155

**Comment by spastorclovr** [comment]
Hey @chrislusf,
Do you have an ETA for a new release including #155? It is pretty much a blocker right now if we wanna add it as a Helm dependency.

Thx :pray:

**Comment by chrislusf** [comment]
released v1.1.9
